flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecPythonCalc.java

```diff
@@ -39,6 +39,14 @@
 @ExecNodeMetadata(
         name = "stream-exec-python-calc",
         version = 1,
+        consumedOptions = {
```

Comment:
I don't have enough knowledge to judge Python. But `python.metric.enabled` sounds like something that does not affect the topology? Maybe skip Python options all together?

Reply:
Also `statebackend.useManagedMemory` sounds like something that can be changed at any time without affecting the topology but only performance?